be/src/main/java/issuetracker/be/controller/UserController.java

```diff
@@ -22,7 +25,12 @@ public UserController(UserService userService) {
   }
 
   @GetMapping("/user")
-  public List<User> showAssignees() {
+  public List<UserResponse> showAssignees() {
     return userService.getAllUsers();
   }
-}
+
+  @PatchMapping("/issue/assignee")
+  public void updateAssignee(@RequestBody IssueAssigneeUpdateRequest issueAssigneeUpdateRequest) {
+    userService.updateAssignee(issueAssigneeUpdateRequest);
+  }
+}
```

Comment:
- assignees 조회는 별도의 표현 없이 `/user` 로 되고
- assignees 변경은 `/issue/assignee` 로 되고 있군요.

Reply:
`/user/assignee` 로 수정해보겠습니다.
```suggestion
@GetMapping("/user")

@PatchMapping("/user/assignee")

```